Composer/packages/server/src/controllers/publisher.ts

```diff
@@ -146,12 +140,12 @@ export const PublishController = {
     if (
       profile &&
       method &&
-      pluginLoader.extensions.publish[method] &&
-      pluginLoader.extensions.publish[method].methods &&
-      pluginLoader.extensions.publish[method].methods.history
+      ExtensionContext.extensions.publish[method] &&
+      ExtensionContext.extensions.publish[method].methods &&
+      ExtensionContext.extensions.publish[method].methods.history
```

Comment:
can this use the same `?.` operator syntax as the `if` statement in `status()` ?